docs/generated-code-usage.md

```diff
@@ -65,3 +65,66 @@ Generated files for Objective-C / C++ are as follows (assuming prefix is `DB`):
+
+Generated files for Python / C++ are as follows:
+
+| Type       | C++ header             | C++ source                   | Python files        | CFFI                | C Wrapper            |
+|------------|------------------------|------------------------------|---------------------|---------------------|----------------------|
+| Enum/Flags | my\_enum.hpp           |                              | my_enum.py          |                     | dh__my_enum.cpp      |
+|            |                        |                              |                     |                     | dh__my_enum.h        |
+|            |                        |                              |                     |                     | dh__my_enum.hpp      |
+| Record     | my\_record[\_base].hpp | my\_record[\_base].cpp :one: | my_record[_base].py |                     | dh__my_record.cpp    |
+|            |                        |                              |                     |                     | dh__my_record.h      |
+|            |                        |                              |                     |                     | dh__my_record.hpp    |
+| Interface  | my\_interface.hpp      | my\_interface.cpp :one:      | my_interface.py     | pycffi_lib_build.py | cw__my_interface.cpp |
+|            |                        |                              |                     |                     | cw__my_interface.h   |
+|            |                        |                              |                     |                     | cw__my_interface.hpp |
+
+- :one: Generated only for types that contain constants.
+
+Additional C Wrapper files are generated for data structures; their names are encoded as:
+
+    dh__{list,set,map}_{encoded_type(s)}.cpp
+    dh__{list,set,map}_{encoded_type(s)}.h
+    dh__{list,set,map}_{encoded_type(s)}.hpp
+
+See the in the table below a few examples:
+
+| Type                    | C Wrapper                              |
+|-------------------------|----------------------------------------|
+| `list<i32>`             | dh__list_int32_t.{cpp,h,hpp}           |
+| `set<string>`           | dh__set_string.{cpp,h,hpp}             |
+| `map<i32, set<string>>` | dh__map_int32_t_set_string.{cpp,h,hpp} |
+
+Add all generated C and C++ source files to your build target, and link it against the
+[djinni-support-lib](https://github.com/cross-language-cpp/djinni-support-lib).
+
+Compile the Python extension module (CFFI) by executing `pycffi_lib_build.py` while providing all C
+Wrapper header files (`.h`) as arguments. The resulting shared library will enable Python to access
+your C++ library through the CFFI bridge.
+
+### Known limitations of the generator
+
+* External types defined in YAML are not yet supported.
+* Use of non-nullable pointers is not yet supported.
+* There are no arguments for configurable header prefixes (or use of existing arguments).  All includes are unqualified.
```

Comment:
I've introduced

- `--c-wrapper-include-prefix <prefix>`
- `--c-wrapper-include-cpp-prefix <prefix>`
- `--c-wrapper-base-lib-include-prefix <prefix>`

So from my point of view header prefixes are as configurable as they can get. Am I missing something?

Reply:
You're right. I didn't realize how many of the limitations we've already addressed so far!